.github/pull_request_template.yml

```diff
@@ -0,0 +1,11 @@
+## Rationale
```

Comment:
I think it would be a good idea to generalize in the bootstrap, yes. At least I realized it was missing here and I find it bothersome to already type the same titles over and over again.
 
But no, the `Issue:` part is not some magic ; we should even probably replace this by `Fix #xxx` since this is what we usually want/use.

Reply:
I've opened https://github.com/openzim/_python-bootstrap/issues/33 so that we can discuss this in a better way:

I suggest that we merge this "like it is" for now, I've just renamed it to .md and replaced the `Issue: ` by a `Fix #xxx`